### ISSUE TITLE
`position` hardcoded?

### ISSUE THREAD

**Issue by huglester** [boxfrommars/rutorika-sortable#15]
Hello,

are there any plans to make it customizable?

**Comment by boxfrommars** [comment]
Done! Available at [v3.0.0](https://github.com/boxfrommars/rutorika-sortable/releases/tag/3.0.0)

Just add `$sortableField` property to your model
```php
class Article extends Model
{
    use \Rutorika\Sortable\SortableTrait;

    protected static $sortableField = 'somefield';
}
```

**Comment by huglester** [comment]
Fast! Thank you!
